splight_lib/models/alert.py

```diff
@@ -51,9 +210,67 @@ class Alert(SplightDatabaseBaseModel):
     description: Optional[str] = Field(
         default=None, max_length=DESCRIPTION_MAX_LENGTH
     )
-    message: str
-    period: float = 1.0
+
+    assets: Optional[QueryFilter] = None
+    alert_items: List[AlertItem]
+    destinations_list: List[str] = []
+
+    type: AlertType
+
     active: bool = True
     status: AlertStatus = AlertStatus.NO_ALERT
-    notification_emails: List[EmailStr] = Field(default_factory=list)
-    conditions: List[AlertCondition] = []
+    status_text: str = ""
+
+    stmt_time_window: int
+    stmt_target_variable: int
+    stmt_operator: StatementOperator
+    stmt_aggregation: StatementAggregation
+    stmt_thresholds: List[AlertThreshold]
+
+    notify_no_data: bool = True
+    notify_error: bool = True
+    notify_timeout: bool = True
+    custom_message: Field(default=None, max_length=200)
```

Comment:
Is the max_length value the same that is used in API?

Reply:
yes